pkg/config/config_test.go

```diff
@@ -641,6 +643,18 @@ var _ = Describe("Config", func() {
 			testConfigs := append(configs, []string{file4, file3, file2, file1}...)
 			gomega.Expect(newConfigs).To(gomega.Equal(testConfigs))
 		})
+
+		It("test config errors", func() {
+			conf := Config{}
+			content := bytes.NewBufferString("")
+			logrus.SetOutput(content)
+			err := readConfigFromFile("testdata/containers_broken.conf", &conf)
+			gomega.Expect(err).To(gomega.BeNil())
+			gomega.Expect(conf.Containers.NetNS).To(gomega.Equal("bridge"))
+			gomega.Expect(conf.Containers.Umask).To(gomega.Equal("0002"))
+			gomega.Expect(content).To(gomega.ContainSubstring("Failed to decode the keys [\\\"foo\\\" \\\"containers.image_default_transport\\\"] from \\\"testdata/containers_broken.conf\\\""))
+			logrus.SetOutput(os.Stdout)
```

Comment:
`os.Stderr`, looking at the implementation.

Or perhaps make `readConfigFromFile` accept a `logrus.Logger` and pass a temporary instance, but that would be notably more disruptive and even more code.